recipes/plotapi/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "plotapi" %}
+{% set version = "6.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 352761ecff3c7cd36c15180af01df50f991c2a223e19b89dbb72306575dde7a0
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.7.1,<4.0.0
+  run:
+    - python >=3.7.1,<4.0.0
+    - requests >=2.20.0,<3.0.0
```

Comment:
Upstream does not declared any dependencies but the code does use requests.

Reply:
Thank you @ocefpaf . I've updated the version to the latest which includes `requests` as a declared dependency.